DEVELOPER.md

```diff
@@ -98,23 +90,17 @@ airos_dfs_reset:
     username: "ubnt"           # root username
     password: "file:/root/pwd" # location to a password file within the core-router
     daytime_limit: "2-7"       # time window for DFS reset
-
-  - name: "segen-mabb"
-    target: "10.31.6.12"
-    username: "ubnt"
-    password: "file:/root/pwd"
-    daytime_limit: "2-7"
 ```
 
 ### network
 
 This part defines IP-Addresses, WIFI-Properties, Mesh and so on. Lets have a (shortened) example of a `networks.yml` from the magda-location:
```

Comment:
we should remove the networks.yml reference here as we now have single file format